docs/README.md

```diff
@@ -202,30 +253,71 @@ Whole px file databases can be validated using ```DatabaseValidator``` class. Va
 Database validation process validates each px file within the database and also the required structure and consistency of the database languages and encoding formats. The return object is a ```ValidationResult``` object that contains ```ValidationFeedback``` objects gathered during the validation process.
 The database needs to contain alias files for each language used in the database for each folder that contains either subcategory folders or px files. If either languages or encoding formats differ between alias or px files, warnings are generated.
    
+##### Example
+```csharp
+	DatabaseValidator validator = new DatabaseValidator("path/to/database");
+	ValidationResult result = validator.Validate();
+	ValidationResult asyncResult = await validator.ValidateAsync(cancellationToken);
+```
+
 ### Computing
 
 ```Matrix<TData>``` class has a set of extension methods for performing basic computations for the datapoints.
 
 #### Sum
 ```SumToNewValue<TData>()``` computes sums of datapoints defined by a subset of values from a given dimension.
 The method takes a new dimension value as a parameter that will define the resulting values.
-The method also has an asyncronous variant ```SumToNewValueAsync<TData>()```.  
+The method also has an asyncronous variant ```SumToNewValueAsync<TData>()```.
+
+##### Example
+```csharp
+    DimensionValue newDimensionValue = new("sumValueCode", new("en", "Sum value")); // New single language dimension value
+	Matrix<DecimalDataValue> output = matrix.SumToNewValue(newDimensionValue, matrix.Metadata.Dimensions[0]); // Sums up all values in the first dimension and creates a new dimension value for the result
```

Comment:
I would remove the comments. Maybe add bried descriptions of the matrix before and after?

Reply:
Comments removed, better explanation added to the top level